terraform/dev/variables.tf

```diff
@@ -52,11 +52,22 @@ variable "fusionauth_tenant" {
 variable "fusionauth_backend_application_id" {
   description = "ID of the FusionAuth application that manages authentication to the web-app's backend"
   type        = string
-  default     = "0853df7b-d5eb-48d6-bec8-9ec48533b008"
+  default     = "8048057e-4f77-406a-a77d-2962a81cea21"
```

Comment:
This is just a comment for clarification in case another person reviews this PR as well, since I was confused by this change at first! This is changing the default value for the FusionAuth app ID from local to dev (which makes sense since this config is for dev deployment anyway).